gcp/compute_engine/serverless_neg_https_lb/locals.tf

```diff
@@ -0,0 +1,90 @@
+locals {
+  list_ae_services = [for service in var.app_engine_services : service.service_name]
+  list_ae_regions = [for region in var.app_engine_services : region.region]
+  app_engine_groups = [
```

Comment:
I think we need a unit test on the list output of each of these neg functions

Reply:
https://protraderseco.atlassian.net/browse/PS-239